.github/workflows/check_verilog.yml

```diff
@@ -0,0 +1,95 @@
+name: check-verilog
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  CIRCT_VERSION: 1.72.0
+  CIRCT_ARCHIVE: "circt-full-shared-linux-x64.tar.gz"
+  DEPENDENCIES_DIR: "deps"
+  MLIR_TARGETS_PATH: "lib/cmake/mlir/MLIRTargets.cmake"
+  BUILD_DIR: "build"
+  UTOPIA_EXECUTABLE: "src/umain"
+  MODULE_OUTPUT_PATH: "output.sv"
+  LIBRARY_OUTPUT_PATH: "lib.sv"
+
+jobs:
+  build-and-parse-verilog:
+    strategy:
+      matrix:
```

Comment:
Too general name, should be renamed.

Reply:
Matrix is the Github-provided strategy name.
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-a-matrix-for-your-jobs